### PR TITLE
Units - Custom Material & SMDI for ATACS uniforms

### PR DESCRIPTION
**Pull request by TyroneMF**
- New base classes for Long/Rolled Atacs uniforms
- Changes how reflective parts of the uniform are.

(Combat rvmat on the left, atacs on the right.)

### CODE REVIEW

File: addons/units/data/uniform_combat_atacs.rvmat [hunk 1]
```diff
@@ -0,0 +1,90 @@
+class StageTI
+{
```
Comment (jonpas) [inline addons/units/data/uniform_combat_atacs.rvmat:2]: This file has to be cleaned up to standards.
- Indentation
- Braces position
- Spaces between numbers

File: addons/units/items/base_helpers.hpp [hunk 2]
```diff
@@ -67,6 +67,12 @@ class CLASS(Unit_Combat_LS_Base): B_Soldier_base_F {
     hiddenSelectionsMaterials[] = {QPATHTOF(data\uniform_combat.rvmat)};
 };
 
+// Combat - Long Sleeves - Atacs
+
+class CLASS(Unit_Combat_LS_Atacs_Base): CLASS(Unit_Combat_LS_Base) {
```
Comment (jonpas) [inline addons/units/items/base_helpers.hpp:72]:
```suggestion
class CLASS(Unit_Combat_LS_ATACS_Base): CLASS(Unit_Combat_LS_Base) {
```
And everywhere else.

**Review by jonpas — CHANGES_REQUESTED** [review]

**Review by jonpas — APPROVED** [review]

File: addons/units/items/base_helpers.hpp [hunk 3]
```diff
@@ -67,6 +67,12 @@ class CLASS(Unit_Combat_LS_Base): B_Soldier_base_F {
     hiddenSelectionsMaterials[] = {QPATHTOF(data\uniform_combat.rvmat)};
 };
 
+// Combat - Long Sleeves - Atacs
+
```
Comment (jonpas) [inline addons/units/items/base_helpers.hpp:71]:
```suggestion
```

**Review by jonpas — COMMENTED** [review]
I thought we are doing this for all uniforms?

**Comment by TyroneMF** [comment]
> 
> 
> I thought we are doing this for all uniforms?

I have no idea tbh. Might be on a case by case basis but these are for the ATACS stuff specifically.

**Review by jonpas — APPROVED** [review]
Right, we fixed ambient and defuse in another PR, woops!

**Pull request merged**